### PR TITLE
Topic/solgs/debian 9 issues

### PR DESCRIPTION
**Pull request by isaak**
-- Debain 9 related tweaks
-- fix selection population page link in user profile page

### CODE REVIEW

File: lib/SGN/Controller/solGS/Files.pm [hunk 1]
```diff
@@ -662,9 +663,7 @@ sub create_file_id {
 	$file_id = $sel_prop ? $file_id . '-' . $sel_prop : $file_id;
     }
 
-    my $trait_id = $traits_ids->[0];
-    $file_id = $traits_selection_id ? $file_id . '_traits_' . $traits_selection_id : $file_id . '_trait_' . $trait_id ;
-  
+    $file_id = $traits_selection_id ? $file_id . '_traits_' . $traits_selection_id : $file_id . '_trait_' . @traits_ids[0];
```
Comment (nickmorales) [inline lib/SGN/Controller/solGS/Files.pm]: i'm not sure @array[0] is proper perl, shouldn't it be $array[0]?

Comment (isaak) [reply lib/SGN/Controller/solGS/Files.pm]: ah, good catch, nick!

Comment (isaak) [reply lib/SGN/Controller/solGS/Files.pm]: fixed.

Comment (lukasmueller) [reply lib/SGN/Controller/solGS/Files.pm]: ah, ok, accepted, merged, etc.

**Pull request merged**